gapis/api/vulkan/api/queue.api

```diff
@@ -123,6 +123,7 @@ cmd VkResult vkQueueSubmit(
       }
     }
     read(info.pWaitDstStageMask[0:info.waitSemaphoreCount])
+    read(info.pCommandBuffers[0:info.commandBufferCount])
```

Comment:
This should not be necessary.
`command_buffers[j]` will do the read.
The case that might be a problem is if we run into
```vkErrorInvalidCommandBuffer(command_buffers[j])```

If there is an error in the command buffer we could see this. We should check this, because this change simply hides that other issue.